teachers_digital_platform/jinja2/teachers_digital_platform/base.html

```diff
@@ -28,6 +28,7 @@
     {{ super() }}
 
     <link rel="stylesheet" href="{{ static('tdp/css/tdp.css') }}">
+    <link rel="stylesheet" href="{{ static('tdp/css/crtool.react.css') }}">
```

Comment:
I don't think you need to add this and the JavaScript reference below in this file, because the CRT is using `crt-base.html`. We probably don't want these assets to load on every TDP page.

Reply:
I will remove the both the tdp.css and crtool.react.css.  Plus the tdp.js and crtool.react.js that are also duplicated between the two files, since crt-base.html has all of them.

Reply:
I'm pretty sure that tdp.css and tdp.js need to stay here, since this is the base file for other TDP pages. Only the React ones need to be removed, right?

Reply:
And this is why we have PR reviews.  For some reason it was in my head the only pages we would use also have my react app, but that is not the case for now.